app/views/admin/users/_menu.html.erb

```diff
@@ -1,48 +1,49 @@
-<div class="flex mb-6 items-center">
-  <ul class="nav nav-pills">
-    <li class="nav-item">
-      <%= link_to "All", admin_users_path, class: "nav-link #{'active' if params[:controller] == 'admin/users' && params[:role].blank?}" %>
+<div class="flex justify-between gap-4">
+  <ul class="crayons-navigation crayons-navigation--horizontal">
+    <li>
+      <%= link_to "People", admin_users_path, class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users'}" %>
     </li>
-    <li class="nav-item">
-      <%= link_to "Admins", admin_users_path(role: :admin), class: "nav-link #{'active' if params[:controller] == 'admin/users' && params[:role] == 'admin'}" %>
+    <li>
+      <%= link_to "Invitations", admin_invitations_path, class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/invitations'}" %>
```

Comment:
Can we also set the `aria-current="page"` attribute when we set `crayons-navigation__item--current`? 

We could even update the CSS so we don't need `crayons-navigation__item--current` and rely on `[aria-current="page"]`instead, to save needing both (but maybe that's a bigger thing and not for now 😄 )

Reply:
I'm adding aria-current attribute but gonna leave adding that extra selector to CSS as it's just a separate thing for a separate PR I think.